cityofaustin/atd-moped
# Refine the tables being tracked in the Moped activity log

## Associated issues

This PR intends to close https://github.com/cityofaustin/atd-data-tech/issues/8055.

## Testing

https://moped-test.austinmobility.io/moped/projects

Testing for this PR is constrained to the `test` instance. Local deployments will lack the AWS based infrastructure. 

**Steps to test:**
Deploy, make any change you can come up with to a project and observe that the changes are reflected in the activity log.

## Deployment considerations
* These changes require a file to be deployed to S3 that maps database tables to primary keys. It needs to be swapped out in conjunction with the deployment of this code.

---
#### Ship list
- [x] Add S3 file operation to deployment checklist
